javascript/ql/src/Security/CWE-601/ServerSideUrlRedirect.qhelp

```diff
@@ -32,6 +36,21 @@ before doing the redirection:
 </p>
 
 <sample src="examples/ServerSideUrlRedirectGood.js"/>
+
+<p>
+Alternatively, we can check that the target URL does not redirect to a different host
+by parsing it relative to a base URL with a known host and verifying that the host
+stays the same:
+</p>
+
+<sample src="examples/ServerSideUrlRedirectGood2.js"/>
+
+<p>
+Note that as written, the above code will allow redirects to URLs on <code>example.com</code>,
+which is harmless but perhaps not intended. Substitute your own domain name for
+<code>example.com</code> to prevent this.
```

Comment:
I think this is more difficult to miss as a comment in the example, but we may want to keep this too. Perhaps more like:

```suggestion
Note that the example code allows redirects to URLs on the test site <code>example.com</code>. You should substitute your own domain name for
<code>example.com</code>.
```

Reply:
OK, I'll adopt a slightly tweaked version. example.com is a real, maintained domain for "illustrative purposes", so I think in practice not changing the example would be fine (or at least wouldn't introduce a security vulnerability), but of course it's preferable for people to substitute their own domain.